core/src/banking_stage.rs

```diff
@@ -514,9 +543,10 @@ impl BankingStage {
 
         let tx_len = verified_transactions.len();
 
-        let (processed, unprocessed_txs) = f(bank, &verified_transactions, poh)?;
+        let (processed, unprocessed_txs) = fn_proc(bank, &verified_transactions, poh)?;
```

Comment:
This code looks unnecessarily awkward. It looks like breaking it up would make it possible to write without closures and minimal copypasta. What if we moved the code above into its own function? Then the caller would invoke that new function and use it to create `(processed, unprocessed_txs)` and `valid_unprocessed_txs` directly.

Reply:
The latest patch should untangle this function.